libs/designsystem/src/lib/components/radio/radio.component.scss

```
@@ -6,26 +6,32 @@ $radio-icon-mark-size: 60%;
 $spacing-to-edge: map-get($checkbox-radio-spacing, 'to-edge');
 $spacing-to-label: map-get($checkbox-radio-spacing, 'to-label');
 
+@mixin wrapper-top-bottom-padding($target-height) {
+  $padding: calc((#{$target-height} - #{$radio-icon-size}) / 2);
```

Do you need to calculate this at runtime? If not, consider replacing with a built time calc:
```suggestion
  $padding: ($target-height - $radio-icon-size) / 2;
```

---

Good point! - Should be fixed by d28acfae53c7e90e4c133f64bb70af305339fa72